### PR TITLE
Fix the workflows

### PR DESCRIPTION
**Pull request by supl**
This PR fixes a bug in the workflows.

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -14,11 +14,13 @@ jobs:
         working-directory: java
 
     steps:
+      - uses: actions/checkout@v2
```
Comment (supl) [inline .github/workflows/docker.yml:17]: This is a stupid mistake.
I forgot to do checkout the code.

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -14,11 +14,13 @@ jobs:
...
+
       - name: Set version
         id: version
         run: |
           VERSION=$(echo ${{ github.ref }} | sed -e "s#refs/tags/v##g")
-          echo ::set-output name=version::${VERSION}
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
```
Comment (supl) [inline .github/workflows/docker.yml:23]: the set-output command will be deprecated on 2023/05/31.
Use the environment variable to replace it.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**